lib/elixir/lib/exception.ex

```diff
@@ -1342,20 +1342,31 @@ defmodule UndefinedFunctionError do
       hint_for_loaded_module(module, function, arity, nil)
   end
 
+  @max_suggestions 10
```

Comment:
Maybe 10 is a lot? How about 5 like in `did_you_mean`?

Reply:
Sure, no strong feelings there